bitnami/valkey-cluster/templates/valkey-statefulset.yaml

```
@@ -20,7 +20,7 @@ spec:
   {{- $podLabels := include "common.tplvalues.merge" ( dict "values" ( list .Values.valkey.podLabels .Values.commonLabels ) "context" . ) }}
   selector:
     matchLabels: {{- include "common.labels.matchLabels" ( dict "customLabels" $podLabels "context" $ ) | nindent 6 }}
-  replicas: {{ .Values.cluster.nodes }}
+  replicas: {{ if eq (typeOf .Values.cluster.nodes) "float64" -}}{{ .Values.cluster.nodes }}{{- else -}}{{ tpl .Values.cluster.nodes . | int }}{{- end }}
```

I'm sorry but I'm not understanding, why the `typeOf` is required? Can't this be simplified like this?

```suggestion
replicas: {{ tpl .Values.cluster.nodes . | int }}
```

For those values that need to be quoted, like environment variables values: {{ tpl .Values.cluster.nodes $ | int | quote }}